src/PkgTaskListWidget.cc

```diff
@@ -123,12 +120,13 @@ bool PkgTaskListWidgetItem::sortByInsertionSequence() const
     if ( ! listWidget() )
         return false;
 
-    PkgTaskListWidget * parentPkgTaskListWidget =
+    const PkgTaskListWidget * parentPkgTaskListWidget =
         dynamic_cast<PkgTaskListWidget *> ( listWidget() );
 
     if ( parentPkgTaskListWidget )
-	return parentPkgTaskListWidget->sortByInsertionSequence();
-    else
-        return false;
+    {
+        return parentPkgTaskListWidget->sortByInsertionSequence();
+    }
+    return false;
```

Comment:
Same as above: The `false` case is not the norm, it's the absolute exception. If somebody gives this function a base class pointer that cannot be dynamic casted, that would be a bug. The code shouldn't crash in that case, but it's clearly not a normal case.